js/src/redux/reducers/facebookEditor.js

```diff
@@ -36,12 +36,22 @@ const facebookReducer = ( state = initialState, action ) => {
 		case SET_FACEBOOK_DESCRIPTION :
 			return { ...state, description: action.description };
 		case SET_FACEBOOK_IMAGE :
-			return { ...state, image: { ...action.image } };
+			return {
+				...state,
+				warnings: action.image.warnings,
+				image: {
+					id: action.image.id,
+					url: action.image.url,
+				},
+			};
 		case CLEAR_FACEBOOK_IMAGE :
-			return { ...state, image: {
-				url: "",
-				id: "",
-			} };
+			return { ...state,
+				image: {
+					url: "",
+					id: "",
+				},
+				warnings: [],
```

Comment:
So, we don't validate content image or featured image?

Same for Twitter including its Facebook fallbacks.

Not saying we should, especially since the warnings are more like suggestions, and the UX could get messy with all fallbacks the customer doesn't know about.